wiki/Beatmapping/Flow/en.md

```diff
@@ -0,0 +1,55 @@
+
+<!-- TODO: this applies to osu!catch as well, the article needs expansion: https://github.com/ppy/osu-wiki/pull/7407#pullrequestreview-988657741 -->
+
+Flow is considered a **subjective** and often opinionated matter due to how different mappers and players design or perceive movement. Some aspects of flow may impact gameplay in different degrees depending on [input device](/wiki/Gameplay/Input_device), such as [graphics tablet](/wiki/Gameplay/Input_device/Graphics_tablet) or [mouse](/wiki/Gameplay/Input_device/Mouse), or the player's acquired skill or experience.
+
+The opposite of flow is called [antiflow](/wiki/Beatmapping/Antiflow), which conversely involves unintuitive or unnatural movement patterns.
+
+## Key components
+
+While movement preferences vary from one person to another, there are a few ideas that give the feeling and impression of flow on a majority of patterns or whole beatmaps.
+
+### Smooth movement
+
+Series of hit objects that require smooth movement, without sharp, abrupt turns, or fast changes of pace, can be considered comfortable to play. In such patterns, the player's cursor may follow a smooth curve throughout. These curves are usually characterised by circular or semicircular segments (so-called *oval* or *circular flow*) going either clockwise or anticlockwise. Beatmaps often switch the direction of the circular flow at regular intervals,[^circular-flow] where the transitions can often accompanied by "neutral" patterns to ease the transition to the next pattern.
+
+Linear movement may also be considered another form of smooth flow, albeit less common, since it does not enforce any specific cursor trajectory type.
+
+### Repetition
+
+Another form of natural movement is repetition. Patterns with similar repeated movements that only differ slightly, such as back-and-forth [jumps](/wiki/Beatmap/Pattern/Jump) or a series of slider pairs, are comfortable to play due to subsequent movements following already familiar paths, making them easy to predict. Repeated patterns may become more and more comfortable throughout a map for the same reason.
+
+### Visual implications
+
+To further make a beatmap look and feel more natural, mappers may use [streams](/wiki/Beatmap/Pattern/Stream) or sliders with curves that reinforce suggested direction of movement. Since sliders bodies do not need to be followed perfectly, because the cursor only needs to reach the [slider ticks](/wiki/Beatmapping/Slider_tick), they may be adjusted and bent in lots of ways without affecting the movement required to clear them. This fact may be used both in favour of an already present movement trajectory, or to help the player pick up a new movement type.
```

Comment:
if this is about slider leniency and the freedom in slider shapes it allows, then what do steams have to do with it? sure stream flow is a thing, but the paragraph continued on to only talk about something specific to sliders

if there's something to say on the visual aspect of streams, then perhaps it can receive its own paragraph

also not entirely sure what the last sentence is about

```suggestion
To further make a beatmap look and feel more natural, mappers may use sliders with different shapes that reinforce suggested direction of movement. Since sliders bodies do not need to be followed perfectly, because the cursor only needs to reach the [slider ticks](/wiki/Beatmapping/Slider_tick), they may be adjusted and bent in lots of ways without affecting the movement required to clear them. This fact may be used both in favour of an already present movement trajectory, or to help the player pick up a new movement type.
```

Reply:
agree w the streams removal
the last sentence is relevant imo. might be better to word as 
`This allows slidershapes to both complement an existing movement trajectory, or help the player pick up a new movement trajectory.`